CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+We adopeted English like official language to commits.
+
+If you want to contribute with this project, follow this steps:
+
+**1 -** [Run the project local](https://github.com/womenwhocoderecife/micropigmentacao-solidaria/README.md)
```

Comment:
Acho que o primeiro passo era ter certeza que está na pasta do projeto

**1 -** Make sure that you're on project folder
```sh
$ pwd
```